app/views/console/users/user.phtml

```diff
@@ -93,6 +93,36 @@
                                 No user preferences found
                             </div>
                         </div>
+
+                        <h3 class="text-danger">Danger Zone</h3>
+
+                        <div class="box danger">
+                            <p>This is the area where you can delete this account.</p>
+                        
+                            <p>By deleting this account you will lose all data associated with this account.</p>
+                        
+                            <p>PLEASE NOTE: Account deletion is irreversible.</p>
```

Comment:
`User` and `Account` in Appwrite mean different things, lets change it to: `User deletion is irreversible`